src/identity/IdentityProviderHttpHandler.ts

```diff
@@ -1,52 +1,68 @@
+import { DataFactory } from 'n3';
+import urljoin from 'url-join';
```

Comment:
Given how often we need it, we might want to re-export this as `joinUrl` from our own path utils.